### DISCUSSION TITLE
兄弟们，我朋友住院了，急需用钱！

### DISCUSSION THREAD

**Discussion by YOULINLING** [share121/inter-knot#256] [其他]
兄弟们，我朋友住院了，急需268或者328。我朋友他有返祖现象，浑身长毛黑毛，还长了尾巴，我要救他，他朋友之前就是浑身红毛，到了晚年就失踪了，我怕他浑身黑毛也会失踪。
![ac41e891f603738d3324491cf51bb051f819ec60 (1)](https://github.com/user-attachments/assets/f7ad6756-728d-49f6-b24f-5f6a26e8ae71)

**Comment by Jiucheng258** [comment]
我敲，这人和我朋友长的好像，而且，我朋友也住院了。哎，希望能有好心的艾丽都市民也来救救我的朋友。

**Comment by vioWistaria** [comment]
你这个朋友是不是明天早上十点准时出院啊

**Comment by warning-Kirov-reporting** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >
我的好兄弟也病了
